wandb/sdk/wandb_settings.py

```diff
@@ -89,6 +91,8 @@
     job_type=None,
     problem=None,
     console=None,
+    offline=None,
+    disabled=None,
```

Comment:
So the new preferred ways to go offline are passing these booleans or setting, WANDB_OFFLINE=true and WANDB_DISABLED=true?  For what it's worth, I liked WANDB_MODE=online,offline,noop

Reply:
ok, I will make them all work and we can decide later :)